spec/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.spec.js

```diff
@@ -0,0 +1,230 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('GOVUK.PIIRemover', function () {
+  var pii
+
+  beforeEach(function () {
+    resetHead()
```

Comment:
Does `resetHead` need to be called before every test if it's already called after every test? That seems like it might be unnecessary.